paymentsheet/src/main/java/com/stripe/android/customersheet/ui/CustomerSheetScreen.kt

```diff
@@ -171,6 +171,7 @@ internal fun SelectPaymentMethod(
                 mandateText = viewState.mandateText,
                 modifier = Modifier
                     .fillMaxWidth()
+                    .padding(top = 8.dp)
```

Comment:
This was previously inside `MandateText`. In a future pull request, we should remove these one-off vertical paddings and have them dictated by the container.